data/campaigns/Secrets_of_the_Ancients/scenarios/15_Mountain_Pass.cfg

```diff
@@ -22,7 +22,7 @@
             #po: "consumption" is an old word for tuberculosis
             story= _ "6 V, 23 YW
 
-We have reached a pass through the mountains that Ras-Tabahn knew of. The air is cool in these upper reaches. Mist rises from the snow in the afternoon, and the icicles drip the purest water from rock overhangs. I find it all refreshing, but Ras-Tabahn’s coughing fits are frequent. Sometimes he is reduced to speaking in short phrases due to lack of breath. Of course he is quite old — 50 at least — but I am also certain that he is ill. His thinness leads me to believe it is consumption, but he refuses to discuss it, preferring to deny the severity of his problem."
+We have reached a pass through the mountains that Ras-Tabahn knew of. The air is cool in these upper reaches. Mist rises from the snow in the afternoon, and the icicles drip the purest water from rock overhangs. I find it all refreshing, but Ras-Tabahn’s coughing fits are frequent. Sometimes he is reduced to speaking in short phrases due to lack of breath. Of course, he is quite old, fifty at least, but I am also certain that he is ill. His thinness leads me to believe it is consumption, but he refuses to discuss it, preferring to deny the severity of his problem."
```

Comment:
I think I prefer this with the dashes. The added comma after "of course" and writing out "fifty" in full are good though.

Reply:
Works for me, and resolves what I mentioned earlier in the general comments.

Reply:
Agree with celmin.